### PR TITLE
Change all paths to absolute #52

### PR DESCRIPTION
**Pull request by IsidoreHouaga**
@janaobsteter would a change like this be what we need for this issue?

### CODE REVIEW

File: Snakemake/workflow/rules/ancestral_inference.smk [hunk 1]
```diff
@@ -7,10 +7,10 @@ if config['ancestral_allele'] == None:
...
             vcf=config['raw_vcf']
         output:
-            info=f"../{Project}/AncestralAllele/Raw_vcf_info.INFO", 
-            log=temp(f"../{Project}/AncestralAllele/Raw_vcf_info.log")
+            info=f"{workdir}/AncestralAllele/RawVcfInfo.INFO",
+            log=temp(f"{workdir}/AncestralAllele/RawVcfInfo.log")
         params:
-            prefix=f"../{Project}/AncestralAllele/Raw_vcf_info"
+            prefix=f"{workdir}/AncestralAllele/RawVcfInfo"
         conda: "bcftools"
```
Comment (janaobsteter) [inline Snakemake/workflow/rules/ancestral_inference.smk:14]: @IsidoreHouaga , there is a change here in the file names from the previous version - it should be "Raw_vcf_info". Put the paths look good :)

**Comment by IsidoreHouaga** [comment]
Changed "RawVcfInfo" to "Raw_vcf_info" in "ancestral_inference.smk" file
@janaobsteter is that fine? If yes I will do for other files.

**Comment by IsidoreHouaga** [comment]
@janaobsteter did you see my previous pull request? ie Changed "RawVcfInfo" to "Raw_vcf_info" in "ancestral_inference.smk" file. Is that fine? If yes I will do for other files.

**Comment by hannesbecher** [comment]
I think this is now implemented. (Covered by PR #67)

**Pull request closed without merging**